src/js/components/TodoList.js

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { connect } from "react-redux"
+import TodoBlock from "./TodoBlock"
+
+@connect((store) => {
+    return {
+        todoItems: store.todo.todoItems
+    };
+})
+export default class TodoList extends React.Component {
+    displayTodoList() {
+        return this.props.todoItems.map((element, index) =>
```

Comment:
Please define TodoList.propTypes. In above file I noticed that you are using propTypes so please keep convention